apps/site/components/Containers/NavBar/index.module.css

```diff
@@ -12,6 +12,11 @@
     dark:bg-neutral-950;
 }
 
+.searchWrapper {
+  @apply flex
+  flex-grow;
```

Comment:
You need to apply indentation here

Reply:
@AugustinMauroy Thanks! Just did it. Can you check?